sharpd/sharp_vty.c

```diff
@@ -416,37 +416,39 @@ DEFPY (install_seg6_routes,
 	return CMD_SUCCESS;
 }
 
-DEFPY (install_seg6local_routes,
-       install_seg6local_routes_cmd,
-       "sharp install seg6local-routes [vrf NAME$vrf_name]\
+DEFPY(install_seg6local_routes, install_seg6local_routes_cmd,
```

Comment:
Just a bit thing, why you udpate syntax from `DEFPY (` to `DEFPY(` ?

Recent added clis is represented as `DEFPY (`
https://github.com/FRRouting/frr/blame/012d6c2cf5ac137f19aacb6c766e2a2ce2af4d18/sharpd/sharp_vty.c